csun-metalab/nomi
# NAM-401 -- UI: Refactor all Vuex API calls to be done in the actions and not the mutations

# Description
Refactored all axios calls from mutations to actions.
  
# Testing
1. Ensure Course list, Faculty Profile, and Student Profile work and don't produce errors.
2. Ensure all api calls are working.
3. Make sure all api calls are refactored to actions.

# Installation
'Are there any dependencies the reviewer needs to download?'
